src/org/jitsi/impl/neomedia/transform/zrtp/ZRTPTransformEngine.java

```diff
@@ -577,14 +577,6 @@ public synchronized boolean initialize(
         zrtpEngine
             = new ZRtp(zf.getZid(), this, clientIdString, config, mitmMode);
 
-        if (timeoutProvider == null)
```

Comment:
This change is in the right direction but I am worried that we may break auto-sensing. How about we just check here if ZRTP is enabled? In Jigasi this is always false.

    if (autoEnable && timeoutProvider == null)